src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard/ClinicalVariableCard.js

```diff
@@ -0,0 +1,533 @@
+                    selectedSurvivalBins.indexOf(bin.key) === -1)
+                  ? '0.33'
+                  : '1',
+              padding: '2px 3px',
+              position: 'static',
+            }}
+            >
+            {selectedSurvivalLoadingIds.indexOf(bin.key) !== -1
+              ? <SpinnerIcon />
+              : <SurvivalIcon />}
+            <Hidden>add to survival plot</Hidden>
+          </Button>
+        </Tooltip>
+      ),
+    },
+  ))
+);
+
+const getBoxTableData = (data = {}) => (
+  Object.keys(data).length
+    ? sortBy(Object.keys(data), datum => boxTableAllowedStats
+      .indexOf(datum.toLowerCase()))
+      .reduce(
+        (tableData, stat) => (
+          boxTableAllowedStats.includes(stat.toLowerCase())
+            ? tableData.concat({
+              count: parseContinuousValue(data[stat]),
+              stat: boxTableRenamedStats[stat] || stat, // Shows the descriptive label
+            })
+            : tableData
+        ), []
+      )
+    : []
+);
+
+const ClinicalVariableCard = ({
+  boxPlotValues,
+  currentAnalysis,
+  dataBuckets,
+  dataDimension,
+  dispatch,
+  dispatchUpdateClinicalVariable,
+  displayData,
+  fieldName,
+  filters,
+  getContinuousBins,
+  id,
+  openCustomBinModal,
+  overallSurvivalData,
+  plots,
+  qqData,
+  resetBins,
+  binsAreCustom,
+  selectedBins,
+  selectedSurvivalBins,
+  selectedSurvivalData,
+  selectedSurvivalLoadingIds,
+  setId,
+  setQQData,
+  setQQDataIsSet,
+  setSelectedBins,
+  style = {},
+  survivalPlotLoading,
+  theme,
+  totalDocs,
+  updateSelectedSurvivalBins,
+  variable,
+  wrapperId,
+}) => {
+  const tableData = variable.active_chart === 'box'
+    ? getBoxTableData(boxPlotValues)
+    : getTableData(
+      displayData,
+      fieldName,
+      getContinuousBins,
+      selectedBins,
+      selectedSurvivalBins,
+      selectedSurvivalLoadingIds,
+      setId,
+      setSelectedBins,
+      totalDocs,
+      updateSelectedSurvivalBins,
+      variable,
+    );
+
+  const histogramData =
+    variable.active_chart === 'histogram'
+      ? tableData.map(d => ({
+        fullLabel: d.groupName || d.key,
+        label: d.groupName || d.key,
+        tooltip: `${d.key}: ${
+          d.chart_doc_count.toLocaleString()} (${
+          (((d.chart_doc_count || 0) / totalDocs) * 100).toFixed(2)}%)`,
+        value: variable.active_calculation === 'number'
+          ? d.chart_doc_count
+          : (d.chart_doc_count / totalDocs) * 100,
+      }))
+      : [];
+
+  const maxKeyNameLength = (
+    maxBy(histogramData
+      .map(d => d.fullLabel), (item) => item.length) || ''
+  ).length;
+
+  const tsvSubstring = fieldName.replace(/\./g, '-');
+  const cardFilters = getCardFilters(
+    variable.plotTypes, selectedBins, fieldName, filters
+  );
+  const disabledCharts = plotType => isEmpty(tableData) &&
+    plotType !== 'delete';
+
+  return (
+    <Column
+      className="clinical-analysis-categorical-card"
```

Comment:
Remove the categorical here since the className shared by both categorical and continuous card.